### DISCUSSION TITLE
BlueSCSI Pico "Almost" works on LC II/LC 475

### DISCUSSION THREAD

**Discussion by compuser101** [erichelgeson/BlueSCSI-v1#244] [Q&A]
I'm excited to have just received a pre-assembled BlueSCSI Pico.  After working through some initial issues (self:RTFM!), I think it's 90% working.  I'm posing the questions to other users to see if it's "just me" or some known issue.  I've been poking around for answers but it seems this is the best place to ask.
1. Both the LC II and 475 fail to "shut down" on command with the BlueSCSI installed.  The Special->shutdown/restart is simply apparently ignored by the computer.  (maybe a symptom of the barebones system, below)
2. I cannot initialize a 2GB image in SC setup; it always ends with SC setup complaining that there was a verification error.  I was able to initialize a 500MB (a 7.1 installed image downloaded from the net; though it wouldn't boot my 475, so I had to reinstall), and a 1GB image I created with "dd", but not the 2GB.  I eventually downloaded a "blank" 2GB image and that seemed to work after a quick format by the Finder.  
3. It seems to be "glitchy".  On the initialized image, I tried install System 7.1 3 times.  Each time, it would fail with a "disk error" at some point in the install.  (third time was right at the very end after it asked fro the first floppy back... ugh.)  I finally just resigned myself to copying the barebones system folder from the disk tools disk.  It appears to be able to boot from that, but that's a minimal a system folder as you can get.  Normal usage doesn't seem to reveal any "errors", but...
4. I mounted a couple of CD images utilizing the BlueSCSI.ini.  The images seem to mount, but any significant read from the image (e.g. copy files) seems to end in some disk error and the copy aborting.

So the overall experience is that of things not being quite there with the v2/pico.  (I've been using the SCSI2SD board on my LC475 up until today and it works fine with that, so the machine itself is stable.)
Anyone have similar experiences/tips?

**Comment by erichelgeson** [comment]
Hey, sorry to hear you're having an issue - defiantly not the norm. I've used a V2 in my LC II quite a bit without issue. 

First lets move over to the V2 discussions here: https://github.com/BlueSCSI/BlueSCSI-v2 - and take a look at the wiki troubleshooting page - those usually resolve any "weirdness".

**Comment by erichelgeson** [comment]
Moved to v2 repo
